modules/async/async/shared/src/main/scala/free/async.scala

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
@raulraja @AdrianRaFo Now we have like two `async.scala` files, one inside the free package and another one outside. Is this the expected behavior?